server/grpc/core/core.go

```diff
@@ -70,6 +71,15 @@ func (s *Server) ExecuteTask(ctx context.Context, request *coreapi.ExecuteTaskRe
 	if err != nil {
 		return nil, err
 	}
+
+	instances, err := s.sdk.Instance.List(&instancesdk.Filter{ServiceHash: hash})
+	if err != nil {
+		return nil, err
+	}
+	if len(instances) != 1 {
```

Comment:
Is this check necessary? Is there be a situation that two or more instances will be deployed on the same host?